src/CabanaPD_Force.hpp

```diff
@@ -251,19 +251,18 @@ void computeForce( ForceType& force, ParticleType& particles,
 template <class ForceType, class ParticleType, class ParallelType>
 double computeEnergy(
     ForceType&, ParticleType&, const ParallelType&,
-    typename std::enable_if<
-        ( !is_energy_output<typename ParticleType::output_type>::value ),
-        int>::type* = 0 )
+    std::enable_if_t<
```

Comment:
Upon second thought, why SFINAE this method instead of just `if constexpr (std::is_same_v<ParticleType::output_type, OutputEnergy>) inside a single definition?